app/models/assessment.rb

```diff
@@ -95,7 +95,7 @@ def get_reported_symptom_by_name(symptom_name)
   def self.get_symptom_names_for_assessments(assessment_ids)
     threshold_cond_hashes = ReportedCondition.where(type: 'ReportedCondition', assessment_id: assessment_ids).pluck(:threshold_condition_hash)
     condition_ids = ThresholdCondition.where(type: 'ThresholdCondition', threshold_condition_hash: threshold_cond_hashes)
-    Symptom.where(condition_id: condition_ids).pluck(:name)
+    Symptom.where(condition_id: condition_ids).pluck(:name).uniq.sort
```

Comment:
Did you find `uniq` was necessary during testing?

Reply:
I included it for right now to be consistent with the existing functionality, but I haven't tested without just yet. My thinking is that it will be necessary, but not 100% sure yet.

Reply:
After some more testing and reviewing, the `uniq` is definitely necessary. Otherwise this will include all Symptoms for the threshold condition hashes for those assessments, and each assessment creates a set of new Symptoms, so there will be duplicates.